frontend/smart-kickers-game/src/apis/goalManipulation.js

```diff
@@ -1,8 +1,17 @@
 import axios from 'axios';
 import config from '../config';
 
-export function pointsManipulation(teamID, action) {
-  axios.post(`${config.apiBaseUrl}/goal?action=${action}&team=${teamID}`).catch((err) => {
-    alert(err);
-  });
+export async function pointsManipulation(teamID, action) {
```

Comment:
Think about changing the name of the function. Currently, it looks like a directory's name 😄. Functions should be named with a verb that gives an information what the function does. What is more, the current name doesn't inform whether it's an API call that usually starts with `fetch`, `get` or `update` etc. I have one name in mind, but I would like to take a look at your proposals.

Reply:
Done. Changed to `updateScores`